spec/spec_helper.rb

```diff
@@ -8,9 +18,19 @@
 require "gds_api/test_helpers/search"
 require "gds_api/test_helpers/content_store"
 require "webmock/rspec"
+WebMock.disable_net_connect!(
+  allow_localhost: true,
+  allow: ["chromedriver.storage.googleapis.com"],
+)
 
 Dir[Rails.root.join("spec/support/**/*.rb")].sort.each { |f| require f }
 
+# For integration tests that use capybara, we configure GovukAbTesting
+# accordingly before/after the tests.
+GovukAbTesting.configure do |config|
```

Comment:
If we're removing the AB test stuff later, then we could remove this block too?

Reply:
ah yep, good point, thanks 👍